src/ui/account.js

```diff
@@ -207,7 +208,14 @@ cwc.ui.Account.prototype.request = function(opts, callback) {
   if (!this.authenticated) {
     this.authenticate();
   }
-  var url = new goog.Uri('https://www.googleapis.com' + opts.path);
+
+  var subdomain = 'www';
+  if (opts.subdomain) {
```

Comment:
For security reasons, please add a simple test to allow only "string" values and no javascript code or something like "www.example.org#r=www.".